### PR TITLE
Style Empty State and 404

### PR DESCRIPTION
**Pull request by rsandb**
Update styling to match rest of app

### CODE REVIEW

File: website/src/pages/404.tsx [hunk 1]
```diff
@@ -1,15 +1,37 @@
...
         <meta name="404" content="Sorry, this page doesn't exist." />
       </Head>
-      <main className="flex h-3/4 items-center justify-center overflow-hidden subpixel-antialiased text-xl">
-        <p>Sorry, the page you are looking for does not exist.</p>
-      </main>
+      <Center bg={backgroundColor2} flexDirection="column" gap="4" fontSize="lg" className="subpixel-antialiased">
+        <Box bg={backgroundColor} p="10" borderRadius="xl" shadow="base">
+          <Box display="flex" flexDirection="column" alignItems="center" gap="8">
+            <FiAlertTriangle size="30" color="DarkOrange" />
+            <Box display="flex" flexDirection="column" alignItems="center" gap="3">
```
Comment (AbdBarho) [inline website/src/pages/404.tsx:22]: would make sense to have `EmptyState` accept children so you can re-use it here?

File: website/src/pages/404.tsx [hunk 2]
```diff
@@ -1,15 +1,37 @@
...
+              <Text>Sorry, the page you are looking for does not exist.</Text>
```
Comment (fozziethebeat) [inline website/src/pages/404.tsx:23]: Can you merge in my changes? I added in a button to file a bug here.

File: website/src/pages/404.tsx (hunk 1 above)
Comment (fozziethebeat) [reply website/src/pages/404.tsx:22]: I agree, `EmptyState` should be a more generic component that takes in children.

**Review by fozziethebeat — CHANGES_REQUESTED** [review]

**Comment by fozziethebeat** [comment]
Also, if you're feeling motivated, I added a `500.tsx` page.  That could use the same styling update as I copied it from `404`.

**Comment by rsandb** [comment]
> Also, if you're feeling motivated, I added a `500.tsx` page. That could use the same styling update as I copied it from `404`.

Is it just in the pages dir? I can't find it

File: website/src/pages/404.tsx (hunk 2 above)
Comment (rsandb) [reply website/src/pages/404.tsx]: working on it now

File: website/src/pages/404.tsx (hunk 1 above)
Comment (fozziethebeat) [reply website/src/pages/404.tsx]: I think this is sufficient for now and a fairly large PR.  my request is to make a follow up PR to make `EmptyState` a re-usable component and update only other components using `EmptyState`.

**Review by fozziethebeat — APPROVED** [review]

**Pull request merged**